css/style.css

```diff
@@ -1,152 +1,116 @@
+/* Reset */
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* Basics */
+
 body {
     background-color: #2e2e2e;
     color: #ddd;
     font-family: "Droid Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
     font-size: 1em;
     line-height: 1.5;
-    overflow-y: scroll;
-}
-
-h1, h2, h3 {
-    color: #fff;
-    font-family: "Mako", Helvetica, Arial, sans-serif;
-    line-height: 1.2;
+    overflow-y: auto;
+    max-width: 55rem;
+    margin: 0 auto;
 }
 
-h2 {
-    font-size: 1.8em;
+img {
+    max-width: 100%
 }
 
 a {
-    color: #2b7dbc;
+    color: #52C0FF;
 }
+
 a:hover {
     text-decoration: none;
 }
 
-#main {
-    max-width: 912px;
-    margin: 0 auto;
-    padding: 0;
+/* Typo */
+
+h1, h2, h3, h4, h5, h6 {
+    color: #fff;
+    font-family: "Mako", Helvetica, Arial, sans-serif;
+    line-height: 1.2;
 }
 
-#nav {
-    float: right;
-    color: #FFF;
-    margin: 10px 18px 0 0;
-    padding: 0;
+/*
+ * Default template styling
+ *  - header
+ *  - nav
+ *  - main
+ *  - #content (legacy)
+ *  - footer
+ */
+
+/* Header */
+
+header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 1rem;
 }
 
-#nav li {
-    list-style: none;
-    float: left;
-    margin-top: 1.8em;
-    margin-left: 5px;
-    border-right: 2px solid #3A8ECD;
+.logo {
+    max-width: 20rem;
 }
 
-#nav li:last-child {
-    border-right: 0;
+/* Nav */
+
+nav ul {
+    display: flex;
 }
 
-#nav li a {
-    margin: 0.4em 0.4em;
-    display: block;
+nav ul li {
+    list-style: none;
+}
+
+nav ul li a {
     color: #FFF;
+    padding: .4rem .8rem;
     text-decoration: none;
     border-bottom: 2px solid #2e2e2e;
+    transition: ease 150ms;
```

Comment:
Can you remove all animations/transitions please?

i3 itself doesn’t have any animations/transitions itself, so it seems weird to add some here.

Reply:
6f38a84